### DISCUSSION TITLE
Is it possible not to resurrect old sessions when opening them after deleting them?

### DISCUSSION THREAD

**Discussion by nothingnesses** [zellij-org/zellij#3108] [Q&A]
When I create a session with `zellij a -c session_name` when there are no other existing sessions, then exit out of it and delete it with `zellij delete-all-sessions`, then recreate it again with  `zellij a -c session_name`, zellij resurrects it, along with all the tabs and processes I'd been running in it before I deleted it. Is it possible to have an option to have zellij create a brand-new, clean session instead of keeping all the old tabs and processes when I do this?

**Comment by imsnif** [comment]
Hey, this is not supposed to happen (and I can't reproduce it myself). Want to maybe make sure there are no typos and such?

Otherwise, this sounds like a bug - but since I can't reproduce it it might be something more particular (eg. corrupt cache, write permission issues, etc.)... if you'd like to investigate further and open a bug with a minimal reproduction that would be cool.

**Reply by nothingnesses** [reply]
After reading your comment, I decided to try removing the zellij cache and now I can't seem to reproduce the issue, so it looks like that fixed it. Thanks!
